data/mods/ssb/moves.js

```diff
@@ -939,7 +939,8 @@ let BattleMovedex = {
 		isNonstandard: "Custom",
 		pp: 5,
 		priority: 0,
-		flags: {dance: 1},
+		flags: {dance: 1, heal: 1},
+		heal: [1, 3],
```

Comment:
causes the move to fail if the user has full HP

Reply:
Thank you. I've been wondering if there was something wrong with that. I'll fix right away.